### ISSUE TITLE
Didnt kill final boss but I can advance to the next stage

### ISSUE THREAD

**Issue by George-Krause** [ZhengPeiRu21/mod-individual-progression#420]
I am a bit confused by the mechanisms of the module. I didnt kill ragnaros/onyxia for the first stage but I can proceed to Blackwing Lair. Why is this possible? I tried to disable the admin account (by creation a new fresh account). But I can go ahead to BWL too without taking the first two bosses.

What am I missing?

**Comment by George-Krause** [comment]
Thats what I am talking about. I havent seen him - didnt touch him and can still access BWL. Does the sentence "Players are able to enter without having defeated Onyxia" mean without killing ragnaros too?

**Comment by Grimfeather** [comment]
![Image1](https://github.com/user-attachments/assets/33979d5d-0c04-4773-bebc-abddd3793ac3)


ok, so let's try to figure this out.
I just tested with a character that is in progression phase 0.
I can't enter BWL through the portal inside LBRS.
I then completed the Blackhand's Command quest.
I still can't enter BWL through the portal inside LBRS
I then went to the orb outside BRS.
Placing my hand on the orb did nothing.

![Image2](https://github.com/user-attachments/assets/e265722d-3044-4d4d-8392-198065b37938)

I can't enter BWL, unless I turn on GM mode.

What is your progression level? it should be 0
